loom/common/core/performance.cpp

```diff
@@ -233,14 +233,6 @@ void LoomProfiler::reset()
 
 LoomProfilerRoot::LoomProfilerRoot(const char *name)
 {
-    for (LoomProfilerRoot *walk = sRootList; walk; walk = walk->mNextRoot)
```

Comment:
Hmm, you're right, this will have to be handled. Maybe we can take advantage of `stringtable_insert` or, a probably better idea, just use a `utHashTable` to keep track of all the roots. Might make things faster as well.